code/src/App.js

```diff
@@ -1,9 +1,100 @@
-import React from 'react';
+/* eslint no-underscore-dangle: 0 */
+import React, { useState, useEffect } from 'react';
+import ThoughtList from 'components/ThoughtList';
+import ThoughtForm from 'components/ThoughtForm';
+import Header from 'components/Header';
 
 export const App = () => {
+  const [thoughtList, setThoughtList] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [newThought, setNewThought] = useState('');
+
+  const fetchThoughts = () => {
+    setLoading(true);
+    fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts')
+      .then((res) => {
+        return res.json();
+      })
+      .then((data) => {
+        setThoughtList(data);
+      })
+      .catch((error) => console.error(error))
+      .finally(() => setLoading(false));
+  }
+
+  useEffect(() => {
+    fetchThoughts();
+  }, []);
+
+  /* eslint-disable no-unused-vars */
+  const handleNewThoughtChange = (event) => {
+    setNewThought(event.target.value);
+  }
+
+  const handleFormCleanup = () => {
```

Comment:
Having a separate function for form-cleanup (rather than packing it into e.g. onFormSubmit) feels very neat and organized.